### DISCUSSION TITLE
Manifestation statements

### DISCUSSION THREAD

**Discussion by pan-zhuo** [uwlib-cams/sinopia_maps#89] [come back to it]
What is the difference between a Manifestation statement element and its non-Manifestation counterpart, when you use unstructured descriptions for both?
For example, how is the value of a manifestation publication statement going to look any different than an unstructured publication statement with the same transcription standard?

Should Manifestation statements be implemented? Will it be confusing to include both Manifestation statements and non-Manifestation statements in a resource template?

**Comment by briesenberg07** [comment]
# manifestation statement and its narrower elements

@pan-zhuo 

Looking at the difference between [publication statement](https://access.rdatoolkit.org/en-US_ala-0f7b5665-dec5-371a-8b7b-d49f1c5080a7) and [manifestation publication statement](https://access.rdatoolkit.org/en-US_ala-8c8972d8-ccd3-3dfd-a06e-7eeb52a996a3) in the Toolkit, I can't come up with any examples for which I'd need a **manifestation publication statement** in addition to or instead of a **publication statement**. (This may be due to my lack of cataloging experience.) But perhaps the choice between these two elements isn't so hard?

*However*, as I think you indicate above, manifestation statement has many [narrower elements](https://access.rdatoolkit.org/en-US_ala-f6ea41ae-fe52-33bf-a7f1-5d0eb5767677/1c638154-d358-4d6a-8c0f-5cb823d19fce), and we aren't sure when or if to use these.

For most all of the "manifestation [...] statement" elements I see prerecording guidance that suggests (to me) use of a "[...] statement" element with domain Manifestation, and/or other Manifestation elements, instead of the "manifestation [...] statement" element itself. For example:

**Manifestation copyright statement** [has under prerecording](https://access.rdatoolkit.org/en-US_ala-7d88fb17-4269-35ae-9534-e7d3c658f1ce/div_ong_4cz_wgb):

![image](https://user-images.githubusercontent.com/27664966/222590836-3ea1f01c-ac58-47a6-adce-26d1af73cc79.png)

**Manifestation production statement** [has under prerecording](https://access.rdatoolkit.org/en-US_ala-8de6cd72-cf5a-363d-a1fa-774433b0a8b7/23ce7d2f-41d4-492d-a580-814791346336):

![image](https://user-images.githubusercontent.com/27664966/222597887-f8ac5e99-a9e8-44b1-9ab6-662d76cc3525.png)

And so forth, suggesting first the **production statement** instead of **manifestation production statement**, and then a series of other Manifestation elements for more specific information such as date of production and place of production.

*What kind* of information about a manifestation would require use of one of the "manifestation [...] statement" elements, as opposed to the other Manifestation elements suggested in prerecording guidance? I have no examples of such information at the moment, and it seems to me that in most cases other elements are preferable.

**Reply by pan-zhuo** [reply]
Regarding **Copyright date** vs. **Manifestation copyright statement**:
**Copyright date** is soft-deprecated, and the recommended approach is listed in [Recording an unstructured description](https://access.rdatoolkit.org/en-US_ala-452cb3af-3c8e-3c20-8d59-2362ad325a09/49e2505d-e20b-443b-9d7b-0657f26d6604)

<img width="428" alt="image" src="https://user-images.githubusercontent.com/100716525/222641863-47e6424b-b4ca-41a7-85f4-c4890669489c.png">

**Reply by briesenberg07** [reply]
Soft-deprecated elements are [given at the RDA Registry](https://www.rdaregistry.info/Aligns/alignSoft2Rec.html), but this doesn't seem to be indicated in the Toolkit.

**Reply by briesenberg07** [reply]
In this specific case, alignment with the LRM dictates that copyright would be at the expression level.

**Comment by pan-zhuo** [comment]
I found this presentation slide that provides some uses for Manifestation statements
Slide 26, http://www.rda-rsc.org/sites/all/files/Recording%20methods%20etc-EN.pdf

<img width="590" alt="image" src="https://user-images.githubusercontent.com/100716525/222645682-4585772b-27bf-4348-82a3-b92bc86fe0a1.png">

**Comment by briesenberg07** [comment]
**☝🏻 PROPOSAL 2023-03-03:**

Include *for now*, test templates, be aware that we may not need these elements. Evaluate length of templates and whether, if "manifestation [...] statement" elements are not used or not used much, they can be removed.

\#comebacktoitlater
